pkg/module/filter.go

```diff
@@ -168,7 +172,7 @@ func newRule(r FilterRule) ruleNode {
 }
 
 func getConfigLines() ([]string, error) {
-	configName := env.IncludeExcludeFileName()
+	configName := env.FilterConfigurationFileName()
```

Comment:
let's include `const op errors.Op = "module.getConfigLines"` and return `errors.E` in the rest of the body of this function :)